tensorrt_llm/serve/openai_server.py

```
@@ -1836,6 +1845,92 @@ async def openai_responses_delete_response(
+            return JSONResponse(content={
+                "success": False,
+                "message": msg
+            },
+                                status_code=500)
+        except Exception as e:
+            logger.error(f"/start_profile failed: {e}")
+            return JSONResponse(content={
+                "success": False,
+                "message": str(e)
+            },
+                                status_code=500)
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)